README.md

```diff
@@ -18,9 +18,11 @@ Add `bootsnap` to your `Gemfile`:
 gem 'bootsnap', require: false
 ```
 
-If you are using Rails, add this to `config/boot.rb` immediately after `require 'bundler/setup'`:
+If you are using Rails, add this to `config/boot.rb` (and remove `bundler/setup` if you have it):
```

Comment:
I think this might be a bit confusing to people. Why should they remove bundler/setup?

Maybe this:

If you are using Rails, add this to `config/boot.rb` immediately after `require 'bundler/setup'` or a `Bundler.setup` call:

```ruby
 require 'bundler/setup'
 require 'bootsnap/setup'
 ```

```ruby
require 'bundler'
Bundler.setup(:default, ENV.fetch('RAILS_ENV', :development)) # Adds only the necessary gems to $LOAD_PATH
require 'bootsnap/setup'
 ```

Protip: Using `Bundler.setup` like in the example above can help boot time by only loading gems from your Gemfile.

Reply:
Coming back at this a few months later so bare with me... 😄 

Unless I'm misunderstanding something, you want to remove `bundler/setup` because otherwise it'll run the bare `Bundler.setup` again (https://github.com/bundler/bundler/blob/master/lib/bundler/setup.rb) without the group setting. Their recommendation is outlined here: http://bundler.io/v1.16/guides/bundler_setup.html. So really, I'd amend my commit to be
```ruby
require 'bundler'
Bundler.setup(:default, :ci)
```

Reply:
@jules2689 just a bump on this?

Reply:
Ah Sorry! This must have gotten lost while I was moving!
I'll look now

Reply:
Oh yea, sorry I wasn't clear.
I'd would like to mention why we want to remove bundler/setup